src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String CSV_SEPARATOR = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        List<String> rawData = readRawData(fromFileName);
+        int[] reportData = calculateReport(rawData);
+        writeReportData(toFileName, reportData);
+    }
+
+    private List<String> readRawData(String fromFileName) {
+        List<String> rawData;
+        try {
+            rawData = Files.readAllLines(Path.of(fromFileName));
```

Comment:
```suggestion
            return Files.readAllLines(Path.of(fromFileName));
```
в такому випадку можна одразу результат повертати

Reply:
Зрозуміло, дякую!
Питання щодо констант. Чи є сенс створювати константи для літералів "supply", "buy" і "result". Наприклад, міняється мова звіту, стандарти назв операцій всередині компанії тощо. Дякую!